kbase-extension/static/kbase/js/common/parameterSpec.js

```diff
@@ -150,7 +147,7 @@ define([], () => {
                 case 'float':
                     return parseFloat(_defaultValue);
                 case 'boolean':
-                    return coerceToBoolean(_defaultValue);
```

Comment:
method does not exist

Reply:
That's wild. Maybe in a util module?
Maybe this one doesn't even get used? I wonder if it's cruft.